atd-vze/src/views/Crashes/RelatedRecordsTable.js

```diff
@@ -88,11 +89,18 @@ const RelatedRecordsTable = ({
       <Table responsive>
```

Comment:
outside the scope of this PR, but it would be nice to see this return/render broken into smaller pieces—e.g. maybe a separate component for the `<form>`...